.travis.yml

```diff
@@ -1,28 +1,7 @@
 language: node_js
+
 node_js:
-  - "8"
-  - "7"
-  - "6"
-  - "5"
-  - "4"
-  - "0.12"
-branches:
-  only:
-    - master
-    - /^greenkeeper/.*$/
-notifications:
-  email:
-    - r@va.gg
```

Comment:
I assume all these people were ignoring the emails?

Reply:
I'm just using the same settings like for all other projects.